sql/src/main/java/org/apache/druid/sql/calcite/planner/Calcites.java

```diff
@@ -82,12 +83,18 @@
 
   private static final DateTimeFormatter CALCITE_TIME_PRINTER = DateTimeFormat.forPattern("HH:mm:ss.S");
   private static final DateTimeFormatter CALCITE_DATE_PRINTER = DateTimeFormat.forPattern("yyyy-MM-dd");
-  private static final DateTimeFormatter CALCITE_TIMESTAMP_PRINTER = DateTimeFormat.forPattern("yyyy-MM-dd HH:mm:ss.S");
+  private static final DateTimeFormatter CALCITE_TIMESTAMP_PRINTER =
+      DateTimeFormat.forPattern("yyyy-MM-dd HH:mm:ss.SSS");
 
   private static final Charset DEFAULT_CHARSET = Charset.forName(ConversionUtil.NATIVE_UTF16_CHARSET_NAME);
 
   private static final Pattern TRAILING_ZEROS = Pattern.compile("\\.?0+$");
 
+  /**
+   * Milliseconds precision.
+   */
+  private final static int TIMESTAMP_LITERAL_PRECISION = 3;
```

Comment:
```suggestion
  private static final int TIMESTAMP_LITERAL_PRECISION = 3;
```

This fixes the checkstyle checking.

Reply:
Thanks!